javav2/example_code/ses/src/test/java/SESTest.java

```diff
@@ -82,61 +80,56 @@ public void whenInitializingAWSService_thenNotNull() {
     @Test
     @Order(2)
     public void SendMessage() {
-
-        try {
-            SendMessage.send(client, sender,recipient, subject,bodyText,bodyHTML);
-            System.out.println("Test 2 passed");
-
-        } catch (IOException | MessagingException e) {
-            e.getStackTrace();
-        }
+        assertDoesNotThrow(() -> SendMessage.send(client, sender,recipient, subject,bodyText,bodyHTML));

```

Comment:
should this pass lintering? Odd spacing of arguments.

Reply:
Fixed